README.md

```diff
@@ -1,2 +1,19 @@
 # PDJ
-# PDJ
+
+Quick-start
```

Comment:
Please use markdown. That's why the file ends with .md, so you can use things like headers and lists.